Traceback (most recent call last):
  File "/home/lei/anaconda3/envs/py36-5/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/lei/anaconda3/envs/py36-5/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/home/lei/compatibility_analysis/pytorch/1.0/graphSAGE-pytorch/src/main.py", line 3, in <module>
    import torch
ModuleNotFoundError: No module named 'torch'
